CHECKS

```diff
@@ -1,3 +1,4 @@
+# Wait for 30 seconds to allow time for migrations, if necessary
```

Comment:
I'm not 100% sure dokku will like the comments in this file, but I've no way of checking that I can think of